resource/css/rtl.css

```diff
@@ -0,0 +1,700 @@
+}
+
+html[dir="rtl"] .search-result-listing .btn-default {
+  left: 10px;
+}
+
+html[dir="rtl"] .search-options .mCSB_container > li > a > .radio {
+  margin: 0;
+  padding-right: 25px;
+  padding-left: initial;
+}
+
+html[dir="rtl"] .search-options .mCSB_container {
+  text-align: right;
+}
+
+html[dir="rtl"] #alphabetical-menu {
+  margin: 15px 15px 0 0;
+}
+
+html[dir="rtl"] .nav {
+  padding-right:0;
+}
+
+html[dir="rtl"] .group-container {
+  margin: 15px 15px 5px 0;
+}
+
+html[dir="rtl"] li.sub-group {
+  margin-right: 25px;
+  margin-left: initial;
+}
+
+/* scrollbars
+ ****************************/
+html[dir="rtl"] .mCSB_container {
+  margin-left: 15px !important;
+  margin-right: initial;
+}
+
+/* tables
+ ****************************/
+
+/* vocabulary
+ ****************************/
+
+html[dir="rtl"] .count-type {
+  padding-left: 20px;
+  padding-right: initial;
+}
+
+/* alerts
+ ****************************/
+html[dir="rtl"] .page-alert {
+  left: 0;
+}
+
+html[dir="rtl"] .alert-replaced a {
+  margin-left: 5px;
+  margin-right: initial;
+}
+
+html[dir="rtl"] #lang-info > h1 {
+  float: right;
+  margin: -5px 0 0 15px;
+}
+
+/* concept
+ ****************************/
+html[dir="rtl"] .concept-info li, .concept-info ul {
+  padding-right: 0;
+}
+
+html[dir="rtl"] .row > .property-label {
+  float: right;
+  padding-right: 15px;
+  padding-left: 15px;
+}
+
+html[dir="rtl"] .row > .property-value-column {
+  padding-right: 15px;
+  padding-left: 15px;
+}
+
+html[dir="rtl"] .restore-breadcrumbs {
+  float: left;
+}
+
+html[dir="rtl"] .crumb-path {
+  float:right;
+}
+
+html[dir="rtl"] .foundby {
+  margin-right: 225px;
+  margin-left: initial;
+}
+
+html[dir="rtl"] .other-languages > .col-xs-6.versal.replaced {
+  padding-left: 0;
+  padding-right:initial;
+}
+
+html[dir="rtl"] .other-languages.first-of-language > .col-xs-6.versal-pref {
+  padding-left: 0;
+  padding-right: initial;
+}
+
+html[dir="rtl"] .other-languages > td:last-of-type > * {
+  padding-right: 15px;
+  padding-left: initial;
+}
+
+html[dir="rtl"] .shortened-symbol {
+  padding: 0 2px 0 0;
+  left: 10px;
+}
+
+html[dir="rtl"] tr .property-hover {
+  margin-right: 5px;
+  margin-left: initial;
+}
+
+html[dir="rtl"] .subvalue {
+  margin-right: 15px;
+  margin-left: initial;
+}
+
+html[dir="rtl"] .property-divider {
+  margin: -20px 0 0 0;
+}
+
+html[dir="rtl"] #vocab-info .table > tbody > tr > th {
+  padding-right: 0;
+  padding-left: initial;
+  text-align: right;
+}
+
+html[dir="rtl"] .pagination {
+  margin: 0 0 -5px 0;
+  padding: 0 10px 0 0;
+}
+
+html[dir="rtl"] .pagination > li {
+  float:right;
+}
+
+html[dir="rtl"] .wide-content {
+  margin-right: 0px;
+  margin-left: initial;
+}
+
+html[dir="rtl"] span.date-info {
+  float: left;
+}
+
+html[dir="rtl"] .concept-download-links > a {
+ margin-left: 1px;
+ margin-right: initial;
+}
+
+
+/* typeahead.js
+ ***********************************/
+html[dir="rtl"] .tt-dropdown-menu {
+  left: 0 !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/NatLibFi/Skosmos/pullRequest?prid=4951454)